pkg/utils/schema.go

```diff
@@ -79,10 +79,20 @@ func LoadBackupTables(ctx context.Context, reader *metautil.MetaReader) (map[str
 }
 
 // ArchiveSize returns the total size of the backup archive.
-func ArchiveSize(meta *backuppb.BackupMeta) uint64 {
+func ArchiveSize(meta *backuppb.BackupMeta, files []*backuppb.File) uint64 {
 	total := uint64(meta.Size())
+	ContainsFile := func(files []*backuppb.File, file *backuppb.File) bool {
```

Comment:
how about use a files map?

Reply:
I think the number of files will not be so much, so O(n) seems enough?

Reply:
Since rocksdb using level compaction, the number of SST files will be restricted.